TELCODOCS-1365 adding missed parameter per-pod-power-management (#61462)
openshift/openshift-docs: modules/cnf-performance-profile-creator-arguments.adoc

Default: `default`.

| `per-pod-power-management`

> a|Enable Per Pod Power Management. This cannot be used with the ultra-low-latency power-consumption-mode.

Reviewer: nit: should ultra-low-latency be **bold**?

> a|Enable Per Pod Power Management. This cannot be used with the ultra-low-latency power-consumption-mode.

Reviewer 2: ```suggestion
a|Enable per-pod power management. This cannot be used if you configured `ultra-low-latency` for the power consumption mode.
```

Reviewer 2: My suggestion for above, following OpenShift doc guidelines to put parameters in backticks.

> a|Enable per-pod power management. This cannot be used if you configured `ultra-low-latency` for the power consumption mode.

Reviewer 3: ```suggestion
a|Enable per-pod power management. You cannot use this argument if you configured `ultra-low-latency` as the power consumption mode.
```

@kquinn1204 Rephrasing to remove passive voice and ambiguous pronoun.
